policyengine_us/parameters/gov/states/ct/tax/income/rates/head_of_household.yaml

```diff
@@ -7,6 +7,8 @@ metadata:
   reference:
     - title: Connecticut General Statutes, Chapter 229, Sec. 12-700, (9), (B)
       href: https://www.cga.ct.gov/current/pub/chap_229.htm#sec_12-700 
+    - title: 2023 Form CT-1040 Connecticut Resident Income Tax Return Instructions
+      href: https://portal.ct.gov/-/media/DRS/Forms/2023/Income/2023-CT-1040-Instructions_1223.pdf#page=20
     - title: 2022 Form CT-1040 Connecticut Resident Income Tax Return Instructions
       href: https://portal.ct.gov/-/media/DRS/Forms/2022/Income/2022-CT-1040-Instructions_1222.pdf#page=20
```

Comment:
could you take a look here:
https://portal.ct.gov/DRS/Miscellaneous-Taxes/Other-Tax-Page/State-Tax-Developments#:~:text=Specifically%2C%20the%203%25%20rate%20on,couples%20will%20decrease%20to%204.5%25.

In 2023 the rate is the same but in 2024 there are changes, could add this as well while we are here